src/core/public/index.ts

```diff
@@ -73,14 +72,10 @@ export { CoreContext, CoreSystem } from './core_system';
  * https://github.com/Microsoft/web-build-tools/issues/1237
  */
 export interface CoreSetup {
-  /** {@link ChromeSetup} */
-  chrome: ChromeSetup;
```

Comment:
shouldn't `yarn core:acceptApiChanges` fail? I'd expect this file to be removed https://github.com/elastic/kibana/blob/412e0852b389e6ae60dbe40c03f176112c3aac29/docs/development/core/public/kibana-plugin-public.chromesetup.md

Reply:
Good catch. This PR appears to have broken something in that check script (it doesn't fail on master):

```
 info Core public API: checking for changes in API signature...
(node:15427) UnhandledPromiseRejectionWarning: Internal Error: Cannot assign isExternal=true for the symbol React because it was previously registered with isExternal=false

You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.
    at AstSymbolTable._fetchAstSymbol (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/AstSymbolTable.ts:477:13)
    at ExportAnalyzer._fetchAstImport (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/ExportAnalyzer.ts:674:52)
    at ExportAnalyzer._tryMatchImportDeclaration (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/ExportAnalyzer.ts:478:23)
    at ExportAnalyzer.fetchReferencedAstEntity (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/ExportAnalyzer.ts:305:35)
    at AstSymbolTable._analyzeChildTree (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/AstSymbolTable.ts:240:58)
    at AstSymbolTable._analyzeChildTree (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/AstSymbolTable.ts:277:12)
    at AstSymbolTable._analyzeChildTree (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/AstSymbolTable.ts:277:12)
    at AstSymbolTable._analyzeChildTree (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/AstSymbolTable.ts:277:12)
    at AstSymbolTable._analyzeChildTree (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/AstSymbolTable.ts:277:12)
    at AstSymbolTable._analyzeChildTree (/Users/jdover/src/kibana/node_modules/@microsoft/api-extractor/src/analyzer/AstSymbolTable.ts:277:12)
(node:15427) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:15427) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Investigating...

Reply:
Fixed with some help from @skaapgif. Moral of the story: the types for React expose a global `React` type that api-extractor doesn't like. Import React specifically in the OverlayService fixed the issue.